nlu_hyperopt/app.py

```diff
@@ -53,7 +69,14 @@ def worker_function(space):
     best_config = space_eval(search_space, best)
     logger.debug("The best values are: {}".format(best_config))
 
-    data_dir = os.environ.get("DATA_DIRECTORY", "/data")
+    data_dir = os.environ.get("INPUT_DATA_DIRECTORY", "./data")
     with open(os.path.join(data_dir, "template_config.yml")) as f:
         config_yml = f.read().format(**best_config)
         logger.info("The best configuration is: \n{}\n".format(config_yml))
+
+        # ## For github action
```

Comment:
but what if it's not a github action? 
I think we should set a variable somewhere `IS_GITHUB_ACTION`, e.g.
```python
# For github action
if IS_GITHUB_ACTION:
   print_result_for_github_action()
else:
  print(best_config)
```

Reply:
It's now set from action.yml